.github/workflows/e2e-fork.yml

```diff
@@ -0,0 +1,237 @@
+
+##################################################################################
+# 
+# This file is automatically generated using templates. Changes to this file
+# should happen through editing the templates under .action_templates/*
+# Manual edits will be overwritten.
+#
+##################################################################################
+
+name: Run E2E Fork
+on:
+  # template: .action_templates/events/labeled-pull-request-target.yaml
+  pull_request_target:
+    types: [labeled]
+    branches:
+    - master
+    paths-ignore:
+    - docs/**
```

Comment:
The only way a pull request will run on a fork, is if someone with write access labels it.